app/metal-controller-manager/api/v1alpha1/environment_types.go

```diff
@@ -65,6 +71,42 @@ type EnvironmentList struct {
 	Items           []Environment `json:"items"`
 }
 
+// EnvironmentDefaultSpec returns EnvironmentDefault's spec.
+func EnvironmentDefaultSpec(talosRelease, endpoint string) *EnvironmentSpec {
+	return &EnvironmentSpec{
+		Kernel: Kernel{
+			Asset: Asset{
+				URL: fmt.Sprintf("https://github.com/talos-systems/talos/releases/download/%s/vmlinuz-%s", talosRelease, runtime.GOARCH),
```

Comment:
I would rather put `arch` as `amd64` for now, as our support for provisioning `arm64` boxes is pretty weak right now, while it's perfectly fine to provision `amd64` boxes from Sidero running on Raspberry Pi.